src/sonic-host-services/setup.py

```diff
@@ -24,6 +24,7 @@
     ],
     install_requires = [
         'dbus-python',
+        'systemd-python',
```

Comment:
>systemd-python [](http://example.com/codeflow?start=9&length=14)

Do you intend to install python3-systemd package? If not, you intend to install it from pypi implicitly?

Reply:
We used to install systemd-python from pypi, and this PR remove it by mistake:
https://github.com/Azure/sonic-buildimage/pull/8861

Reply:
I see. Prefer explicitly `pip3 install` in `files/build_templates/sonic_debian_extension.j2`.